source/server.js

```diff
@@ -179,8 +178,8 @@ if (config.authentication) {
 
 const indexHtmlCacheKey = cache.registerFunc(() => {
   return cache.resolveFunc(pluginsCacheKey).then((plugins) => {
-    return fs.readFileAsync(__dirname + '/../public/index.html').then((data) => {
-      return Bluebird.all(Object.keys(plugins).map((pluginName) => {
+    return fs.readFile(__dirname + '/../public/index.html').then((data) => {
+      return Promise.all(Object.keys(plugins).map((pluginName) => {
         return plugins[pluginName].compile();
       })).then((results) => {
```

Comment:
I know it's not strictly related, but can we use `Object.values` here, since we don't seem to care about the plugin name.

```suggestion
      return Promise.all(Object.values(plugins).map((plugin) => plugin.compile)).then((results) => {
```